netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -225,6 +226,18 @@ public static SslContextBuilder configure(SslContextBuilder builder, Provider jd
         throw new IllegalArgumentException(
             SUN_PROVIDER_NAME + " selected, but Jetty NPN/ALPN unavailable");
       }
+    } else if (IBM_PROVIDER_NAME.equals(jdkProvider.getName())) {
+      // Jetty ALPN/NPN only supports one of NPN or ALPN
```

Comment:
So the Security Provider here is `IBMJSSE2` and after falling thru to line 248 you are still passing `Http2SecurityUtil.CIPHERS` which according to its documentation at https://netty.io/4.1/api/io/netty/handler/codec/http2/Http2SecurityUtil.html says "The following list is derived from SunJSSE Supported Ciphers and Mozilla Modern Cipher Suites in accordance with the HTTP/2 Specification."

The actual list of ciphers for IBMJSSE2 is listed at https://www.ibm.com/support/knowledgecenter/en/SSYKE2_7.0.0/com.ibm.java.security.component.70.doc/security-component/jsse2Docs/ciphersuites.html . 

May be they match or there is no problem with this code but it will be good to confirm that and may be add a comment to that effect?